home/serializers.py

```diff
@@ -224,43 +227,30 @@ def body_field_representation(page: Any, request: Any) -> Any:
                                 format_whatsapp_template_message(template.message),
                             ),
                             ("is_whatsapp_template", "True"),
-                            ("whatsapp_template_name", template.name),
+                            ("whatsapp_template_slug", template.slug),
```

Comment:
Doesn't this mean that we're changing the v2 API? Shouldn't we keep it `whatsapp_template_name`, and just have it reference `template.slug`?